src/community/index.md.njk

```diff
@@ -1,52 +1,95 @@
 ---
 layout: layout-pane.njk
 title: Community
-description:  The GOV.UK Design System brings together the latest research, design and development from across government to make it sure it’s representative and relevant for its users
+description:  Everyone can help improve the Design System by joining our community discussions, events and co-design collaborations
 show_subnav: true
 ---
 
-The GOV.UK Design System is for everyone. It has a strong cross-government community behind it. It includes the latest research, design and development to represent and be relevant for its users.
+Everyone can help improve the Design System by joining our community discussions, events and co-design collaborations.
 
-Find out:
+We depend on a strong cross-government community to ensure the Design System includes the latest research, design and development to represent and be relevant for its users.
 
-- about [community resources and tools](/community/resources-and-tools/)
-- what people are currently working on in the [community backlog](/community/backlog/)
-- how to [propose a component or pattern](/community/propose-a-component-or-pattern/)
-- how to [develop a component or pattern](/community/develop-a-component-or-pattern/)
-- about [blog posts, videos and podcasts](/community/blogs-talks-podcasts/)
+## Sign up to our mailing list
+[Sign up to get emails about the Design System](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system) to find out when we announce events and workshops.
 
-Learn how the [Design System working group](/community/design-system-working-group/) reviews and approves components and patterns to confirm they meet the [contribution criteria](/community/contribution-criteria/).
+## Join the conversation
 
-## Community principles
+Here’s a few places to join the discussions that help shape the Design System.
 
-These principles are for the community of people that create and contribute to government components and patterns.
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h3>Chat with us on Slack</h3>
+    <img src="/styles/images/default/3by2.jpg" alt="Three Olympic cyclists overlayed with a 3 by 2 grid to show the 3 by 2 ratio.">
```

Comment:
Ensure image assets are optimised for page load time and accessibility — @owenatgov

Reply:
Govuk have some pretty solid content on how they handle the updating of promo slots on the govuk home page https://github.com/alphagov/frontend/blob/main/docs/update-homepage-promotion-slots.md